app/src/main/java/ar/scacchipa/twittercloneapp/presentation/LoginActivity.kt

```diff
@@ -1,10 +1,10 @@
-package ar.scacchipa.twittercloneapp.component
+package ar.scacchipa.twittercloneapp.presentation
 
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
 import ar.scacchipa.twittercloneapp.databinding.ActivityLoginLayoutBinding
 
-class LoginCloneActivity : AppCompatActivity() {
+class LoginActivity : AppCompatActivity() {
```

Comment:
move this to the corresponding login folder (the webview section or the other one), no reason to be outside

Reply:
A new folder level was added to the presentation folder structure.
- / presentation
  - / start
    - / splash
    - / login
  - / main
    - / home
    - / search